src/Extension/Details/Node/DetailsNode.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of the HTML sanitizer project.
+ *
+ * (c) Titouan Galopin <galopintitouan@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace HtmlSanitizer\Extension\Details\Node;
+
+use HtmlSanitizer\Node\AbstractTagNode;
+use HtmlSanitizer\Node\HasChildrenTrait;
+
+/**
+ * @author Titouan Galopin <galopintitouan@gmail.com>
+ *
+ * @final
+ */
+class DetailsNode extends AbstractTagNode
+{
+    use HasChildrenTrait;
+
+    public function getTagName(): string
+    {
+        return 'details';
+    }
+
+    public function render(): string
+    {
+        $isOpen = null !== $this->getAttribute('open');
+
+        return '<details'.($isOpen ? ' open="open"' : '').'>'.$this->renderChildren().'</details>';
```

Comment:
this forbids allowing more attributes on the `details` node, as they would get lost here.

Reply:
Indeed, my mistake. I'll fix that asap tomorrow.